core/server/src/main/webapp/browse-body.jsp

```diff
@@ -38,6 +65,10 @@
               <% } %>
             </ul>
           </div>
+          <div id="pathNav">
```

Comment:
See comment above about moving `<div id="pathNav">` to inside `<div class="navbar-inner">`